main/webapp/modules/core/langs/translation-en.json

```diff
@@ -91,6 +91,8 @@
     "core-index-import/uploading-data": "Uploading data…",
     "core-index-import/web-address": "Web Addresses (URLs)",
     "core-index-import/warning-web-address": "You must specify a valid web address (URL) to import.",
+    "core-index-import/invalid-url":"INVALID URL",
+    "core-index-import/blank-url":"BLANK URL",
```

Comment:
Why fully capitalized? Your screenshot shows a capitalization which seems more consistent with the rest of the tool, no?

Reply:
Sorry, changed it back to the real one, I had it capitalized while testing it